fdbserver/ptxn/TLogPeekCursor.actor.h

```diff
@@ -476,10 +475,10 @@ class BroadcastedStorageTeamPeekCursorBase : public ptxn::details::VersionSubseq
 protected:
 	BroadcastedStorageTeamPeekCursorBase(std::unique_ptr<details::CursorContainerBase>&& pCursorContainer_,
 	                                     std::unique_ptr<details::CursorContainerBase>&& pCursorContainerSnapshot_,
-	                                     const Version& version_ = 0)
+	                                     const Version& version_ = -1)
 	  : pCursorContainer(std::move(pCursorContainer_)),
-	    remoteMoreAvailableSnapshot{ false, version_, std::move(pCursorContainerSnapshot_) }, currentVersion(version_) {
-	}
+	    remoteMoreAvailableSnapshot{ false, version_, std::move(pCursorContainerSnapshot_) },
+	    currentVersion(version_ + 1) {}
```

Comment:
currentVersion should be initialized to version_ -1.

version_ is the first version that this cursor doesn't know, so the currentVersion (the latest known version) should be version_ - 1.

Reply:
I have some updated code for you to review.